pdg/src/info.rs

```diff
@@ -1,11 +1,12 @@
 use crate::graph::{Graph, Node, NodeId, NodeKind};
 use crate::Graphs;
-use rustc_middle::mir::Field;
 use std::cmp::max;
-use std::collections::hash_map::Entry;
 use std::collections::{HashMap, HashSet};
 use std::fmt::{self, Debug, Display, Formatter};
 
+/// Force an import of [`Node`] just for docs.
+const _: Option<Node> = None;
```

Comment:
this can't be resolved in the docstring itself?

Reply:
Yeah, but then we have to add a `[`Node`]: crate::graph::Node` to almost every doc comment.  This does it globally.  The other way works fine, too, but it just adds a ton of extra lines vs. this one line.

Reply:
is there a module-level doc that it could be snuck into, thereby making it available elsewhere, or is that not how rustdoc works?

Reply:
Yeah, unfortunately, `rustdoc` doesn't work that way.  I've tried `#[cfg(doc)] use ...` in the past for doc-only imports, but it didn't work either.  Then I just thought of this as another workaround.